examples/src/main/python/prefetch.py

```diff
@@ -0,0 +1,86 @@
+#
```

Comment:
I think examples in this directory target to show how the feature or API is used rather than showing the perf results .. 
Virtually the example seems it has to be just `.toLocalIterator(prefetchPartitions=False)` which I don't think worth as a separate example file.